docs/run_test/troubleshoot_failures.rst

```diff
@@ -0,0 +1,184 @@
+Troubleshoot Test Failures
+=======================
+
+-  `Overview <#overview>`__
+-  `Test results <#test-results>`__
+-  `Console output <#console-output>`__
```

Comment:
The sections should follow what we recommended in the Overview. Like the "Error Message" section, it explains the test results, and where the error message can be found, and so on.

Reply:
https://mslisa--3528.org.readthedocs.build/en/3528/run_test/troubleshoot_failures.html